yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/CreateFaceCaptureResourceResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.yoti.api.client.docs.session.retrieve;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CreateFaceCaptureResourceResponse {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("frames")
+    private Integer frames;
```

Comment:
Why use an _Integer_ object?

Reply:
> Why use an _Integer_ object?

The primitive `int` will default to 0 if we don't send `frames` back.  Using an object will be `null` if the property is omitted in the response

Reply:
But we do send frames back...